scripts/gen_app_partitions.py

```diff
@@ -34,7 +41,8 @@
 """
 
 library_bss_template = """
-		*{0}:*(.bss .bss.* COMMON COMMON.*)
+		*{0}:*(SORT_BY_ALIGNMENT(.bss) SORT_BY_ALIGNMENT(.bss.*)
+		       SORT_BY_ALIGNMENT(COMMON) SORT_BY_ALIGNMENT(COMMON.*))
```

Comment:
strictly speaking, if I understand correctly this change is orthogonal to this patch and is more related to #14183 can you make this change there instead?